src/status_im/chat/models_test.cljs

```diff
@@ -91,3 +92,8 @@
     (testing "Pagination info should be reset on navigation"
       (let [res (chat/navigate-to-chat-nav2 {:db db} chat-id false)]
         (is (nil? (get-in res [:db :pagination-info chat-id :all-loaded?])))))))
+
+(deftest camera-roll-loading-more
```

Comment:
Just a missing `-test` suffix. `deftest` creates a var, so it's conventional to add a suffix.